src/index/DeltaTriples.h

```diff
@@ -40,8 +40,8 @@ class SharedLocatedTriplesSnapshot
 
 // A class for keeping track of the number of triples of the `DeltaTriples`.
 struct DeltaTriplesCount {
-  size_t triplesInserted_;
-  size_t triplesDeleted_;
+  int64_t triplesInserted_;
+  int64_t triplesDeleted_;
```

Comment:
Can those be negative, or what is the reason here?

Reply:
Ah, when you have less inserted triples after an operation (because you have deleted some), then the diff can be negative. Can you quickly conform, that that is correct?

Reply:
Yes and vice versa. The difference was already returned, so the numbers could underflow.